src/app/models/figure.ts

```diff
@@ -42,16 +42,15 @@ export function createEmptyLicenseAttributes(): Figure {
   };
 }
 
-export function getFigureImageUrl(el: Element): string {
+export function getFigureImageUrlFromXml(el: Element): string {
   const paths = get(el.ownerDocument, 'manuscriptPath').split('/');
   const id = paths[2];
+  const fileName = get(el.querySelector('graphic'), 'attributes.xlink:href.value').replace('tif', 'jpg');
```

Comment:
Both `.tif` and `.tiff` are valid extensions, hence you might need to make this more robust to cop with both cases. As I interpret it, this could result in `.jpgf` if it were to encounter `.tiff`.

Reply:
yeah it's some old code we had from before. I'll update it

Reply:
I've accounted for this on the server too so we should be good to support both.